src/menu.ts

```diff
@@ -675,71 +675,51 @@ export default class Menu extends LitElement {
   }
 
   #onDefaultSlotMouseOver(event: MouseEvent) {
-    // Chrome has this funky Popover API bug where the popover is, for about 10
-    // milliseconds, rendered above where it's supposed to be whenever `showPopover()`
-    // is called.
-    //
-    // It's not clear if the popover is invisible during that time, or if it's visible
-    // and 10 milliseconds isn't enough to be noticable. Either way, it's effectively
-    // invisible but still picks up "mouseover" events.
-    //
-    // If the user's mouse happens to be over one of the Option(s) that's inside our
-    // invisible, mispositioned popover, then the Option is activated. Thus this
-    // variable and the guard below it.
-    //
-    // There's no mention of the above issue in it. But this bug is probably a good one
-    // to keep an eye on: https://issues.chromium.org/issues/364669918.
-    const isOutOfBounds =
-      this.#componentElementRef.value &&
-      event.y < this.#componentElementRef.value.getBoundingClientRect().y;
```

Comment:
Getting rid of the out-of-bounds check means that the above bug will be reintroduced. But that bug was an issue before I recently reworked Menu. So we're no worse off.

I'd fix the bug now if the fix were obvious. Rather than Chromium, Floating UI actually appears to be the cause. So I'll need to do some digging. It's possible Floating UI is misconfigured. It's equally possible it's a Floating UI bug.